src/core/reducers/api.js

```diff
@@ -48,24 +46,6 @@ export const initialApiState: ApiStateType = {
   userId: null,
```

Comment:
`userId` should be removed from `initialApiState`. I think this will break some things. Those things will need to use `state.user.id` instead.

Reply:
doh, bad rebase it seems because I am pretty sure I removed it :x